rust/sealevel/client/src/warp_route.rs

```diff
@@ -255,23 +263,78 @@ impl RouterDeployer<TokenConfig> for WarpRouteDeployer {
             TokenType::Synthetic(_token_metadata) => {
                 let decimals = init.decimals;
 
-                let init_txn = ctx.new_txn().add(
-                    hyperlane_sealevel_token::instruction::init_instruction(
-                        program_id,
-                        ctx.payer_pubkey,
-                        init,
+                ctx
+                    .new_txn()
+                    .add(
+                        hyperlane_sealevel_token::instruction::init_instruction(
+                            program_id,
+                            ctx.payer_pubkey,
+                            init,
+                        )
+                        .unwrap(),
                     )
-                    .unwrap(),
-                );
+                    .with_client(client)
+                    .send_with_payer();
 
                 let (mint_account, _mint_bump) =
                     Pubkey::find_program_address(hyperlane_token_mint_pda_seeds!(), &program_id);
-                // TODO: Also set Metaplex metadata?
-                init_txn.add(
+
+                // the metadata pointer should be initialized here, right before the mint initialization below
+                // but this is via the CLI most likely
+                // `spl-token initialize-metadata [FLAGS] [OPTIONS] <TOKEN_MINT_ADDRESS> <TOKEN_NAME> <TOKEN_SYMBOL> <TOKEN_URI>`
+                // `spl-token update-metadata-address <TOKEN_MINT_ADDRESS> <METADATA_ADDRESS>`
+                let chain_url = client.url();
+                println!("chain_url: {:?}", chain_url);
+                let mut cmd = Command::new(spl_token_binary_path.clone());
+                cmd.args([
+                    "create-token",
+                    mint_account.to_string().as_str(),
+                    "--enable-metadata",
+                    "-p",
+                    "TokenzQdBNbLqP5VEhdkAS6EPFLC1PHnBqCXEpPxuEb",
+                    "--url",
+                    client.url().as_str(),
+                    "--with-compute-unit-limit",
+                    "500000",
+                ]);
+                println!("running command: {:?}", cmd);
+                let status = cmd
+                    .stdout(Stdio::inherit())
+                    .stderr(Stdio::inherit())
+                    .status()
+                    .expect("Failed to run command");
+
+                println!("initialized metadata pointer. Status: {status}");
+
+                let required_rent = client
+                    .get_minimum_balance_for_rent_exemption(SyntheticPlugin::MINT_ACCOUNT_SIZE)
```

Comment:
I thought this was supposed to be the length we expect to have after the metadata initialize? like something closer to 600+ bytes

Iiuc this size is the one that the warp route initialize sets for the mint, which will already make it rent exempt with a transfer from the payer

Reply:
you're right, this is the log from the deployment: `Funding mint account 8e7oRHQ2VVzgjncL8gX75PWigq9q3qfJMgxtA9mEwgfP with funding_amount 0 to reach total balance of 24616`. 

But fortunately the difference in rent is paid in the metadata initialization transaction, so I'll remove this logic entirely: https://explorer.eclipse.xyz/tx/VuT3asE5yr1P6g4rDtCcxUzoG7ZX6aHXpnrobNdrEik14wNKmtK9WFZB8mkVUrsJ1WDzzCG3NkBNSwW9unpgXox